main/src/cgeo/geocaching/settings/Settings.java

```diff
@@ -85,6 +85,16 @@
     @NonNull private static final String TWITTER_KEY_CONSUMER_PUBLIC = CryptUtils.rot13("ESnsCvAv3kEupF1GCR3jGj");
     @NonNull private static final String TWITTER_KEY_CONSUMER_SECRET = CryptUtils.rot13("7vQWceACV9umEjJucmlpFe9FCMZSeqIqfkQ2BnhV9x");
 
+    private static final boolean TRANSPARENT_BACKGROUND_DEFAULT;
+
+    static {
+        if (Build.PRODUCT.startsWith("asop") && Build.VERSION.SDK_INT == 27) {
```

Comment:
I believe this should be "aosp" not "asop"

Reply:
You are so right...